### PR TITLE
Feat/CI-22

### PR DESCRIPTION
**Pull request by BARBUR02**
Added basic layout and landing page based on design basis.

image

### CODE REVIEW

File: frontend/src/components/CodeLanguageList.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import { Badge, Flex, Icon } from '@chakra-ui/react';
+
+// const CodeLanguages = ['Python', 'Javascript', 'Ruby', 'Elixir', 'C++', 'C#'];
```
Comment (socz3qqq) [inline frontend/src/components/CodeLanguageList.tsx]: Unnecessary comment, either delete it or use it with zip function to generate the mappings below

File: frontend/src/components/CodeSmellList.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import { Badge, Flex } from '@chakra-ui/react';
+
+// const CodeSmells = [
+//   'Comment',
+//   'Incorrect typing',
+//   'Typo',
+//   'Logic Error',
+//   'Invocation Error'
+// ];
```
Comment (socz3qqq) [inline frontend/src/components/CodeSmellList.tsx]: Unnecessary comment, either delete it or use it with zip function to generate the mappings below

File: frontend/src/components/CodeLanguageList.tsx (hunk 1 above)
Comment (BARBUR02) [reply frontend/src/components/CodeLanguageList.tsx]: Moved Literal `CodeSmell` and `Language` types to module with quizData for now.

File: frontend/src/components/CodeSmellList.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { Badge, Flex } from '@chakra-ui/react';
+
+const CodeSmellColorMapping = new Map([
+  ['Comment', 'red'],
```
Comment (Fifus17) [inline frontend/src/components/CodeSmellList.tsx]: 'Comment' twice in one map

**Pull request merged**